mm2src/coins/nft/storage/wasm/wasm_storage.rs

```diff
@@ -0,0 +1,235 @@
+        todo!()
+    }
+
+    async fn get_nft_amount(
+        &self,
+        _chain: &Chain,
+        _token_address: String,
+        _token_id: BigDecimal,
+    ) -> MmResult<Option<String>, Self::Error> {
+        todo!()
+    }
+
+    async fn refresh_nft_metadata(&self, _chain: &Chain, _nft: Nft) -> MmResult<(), Self::Error> { todo!() }
+
+    async fn get_last_block_number(&self, _chain: &Chain) -> MmResult<Option<u32>, Self::Error> { todo!() }
+
+    async fn get_last_scanned_block(&self, _chain: &Chain) -> MmResult<Option<u32>, Self::Error> { todo!() }
+
+    async fn update_nft_amount(&self, _chain: &Chain, _nft: Nft, _scanned_block: u64) -> MmResult<(), Self::Error> {
+        todo!()
+    }
+
+    async fn update_nft_amount_and_block_number(&self, _chain: &Chain, _nft: Nft) -> MmResult<(), Self::Error> {
+        todo!()
+    }
+}
+
+#[async_trait]
+impl NftTxHistoryStorageOps for IndexedDbNftStorage {
+    type Error = WasmNftCacheError;
+
+    async fn init(&self, _chain: &Chain) -> MmResult<(), Self::Error> { todo!() }
+
+    async fn is_initialized(&self, _chain: &Chain) -> MmResult<bool, Self::Error> { todo!() }
+
+    async fn get_tx_history(
+        &self,
+        _chains: Vec<Chain>,
+        _max: bool,
+        _limit: usize,
+        _page_number: Option<NonZeroUsize>,
+        _filters: Option<NftTxHistoryFilters>,
+    ) -> MmResult<NftsTransferHistoryList, Self::Error> {
+        todo!()
+    }
+
+    async fn add_txs_to_history<I>(&self, _chain: &Chain, _txs: I) -> MmResult<(), Self::Error>
+    where
+        I: IntoIterator<Item = NftTransferHistory> + Send + 'static,
+        I::IntoIter: Send,
+    {
+        todo!()
+    }
+
+    async fn get_last_block_number(&self, _chain: &Chain) -> MmResult<Option<u32>, Self::Error> { todo!() }
+
+    async fn get_txs_from_block(
+        &self,
+        _chain: &Chain,
+        _from_block: u32,
+    ) -> MmResult<Vec<NftTransferHistory>, Self::Error> {
+        todo!()
+    }
+
+    async fn get_txs_by_token_addr_id(
+        &self,
+        _chain: &Chain,
+        _token_address: String,
+        _token_id: BigDecimal,
+    ) -> MmResult<Vec<NftTransferHistory>, Self::Error> {
+        todo!()
+    }
+
+    async fn get_tx_by_tx_hash(
+        &self,
+        _chain: &Chain,
+        _transaction_hash: String,
+    ) -> MmResult<Option<NftTransferHistory>, Self::Error> {
+        todo!()
+    }
+
+    async fn update_tx_meta_by_hash(&self, _chain: &Chain, _tx: NftTransferHistory) -> MmResult<(), Self::Error> {
+        todo!()
+    }
+
+    async fn update_txs_meta_by_token_addr_id(&self, _chain: &Chain, _tx_meta: TxMeta) -> MmResult<(), Self::Error> {
+        todo!()
+    }
+
+    async fn get_txs_with_empty_meta(&self, _chain: &Chain) -> MmResult<Vec<NftTokenAddrId>, Self::Error> { todo!() }
+}
+
+#[derive(Clone, Debug, Deserialize, Serialize)]
+pub(crate) struct NftListTable {
+    token_address: String,
+    token_id: BeBigUint,
+    chain: String,
```

Comment:
is `chain` here same as `ticker`?

Reply:
No, I use `chain.to_string()` here in the case if I need to get `Chain` type from string.
And actually I believe there is no so big difference between `chain.to_string()` and `chain.to_ticker()`  if we want to use it as the index for instance, bcz both are unique.

Reply:
Ok thanks. but I think if we should use `ticker` if we can for the field name just to be consistent (with other `WASM` storage impls).

Reply:
In sqlite version everywhere I use to_string, except scanned_nft_blocks table, bcz ticker names are shorter, so it was a little easier for me to read rows in this small table (may be there I also should use `to_string(`)). NFT feature is quite different from other implementations, it relies on blockchain type most of the moments, not platform coin ticker.